FortnoxAPILibrary/Connectors/InvoiceConnector.cs

```diff
@@ -299,7 +309,19 @@ public bool NotCompleted
 			}
 		}
 
-		private bool filterBySet = false;
+        /// <summary>
+        /// Use with Find() to limit the search result
+        /// </summary>
+        [FilterProperty("credit")]
+        public bool IsCredit { get; set; }
```

Comment:
`IsCredit` => `Credit` also [FilterProperty("credit")] => [FilterProperty]
IsCredit is inconsistent with all other bool search parameters in repo, for example `Sent`.

Reply:
Can not do like that because of Credit method.

Reply:
Aaah, ok. Hm... Well, I guess leave it as is then.